modules/nw-ingress-sharding.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc
+// * ingress-operator.adoc
+
+[id="nw-ingress-sharding_{context}"]
+= Configuring ingress controller sharding
+
+Ingress Controllers that have permission to view all of the labels in all
+projects can select routes to admit based on the labels. This is called Ingress
+Controller sharding. This is useful when balancing incoming traffic load among a
+set of Ingress Controllers and when isolating traffic to a specific Ingress
+Controller. For example, company A goes to one Ingress Controller and company B
+to another.
+
+.Procedure
+
+. Edit the `router-internal.yaml`:
++
+----
+# cat router-internal.yaml
+apiVersion: v1
+items:
+- apiVersion: operator.openshift.io/v1
+  kind: IngressController
+  metadata:
+    name: sharded
+    namespace: openshift-ingress-operator
+  spec:
+    domain: <apps-sharded.basedomain.example.net>
```

Comment:
Not critical, but maybe a callout or two would be helpful here so users know what in here they need to configure for their specific needs?